internal/storage/postgres/tree/get.go

```diff
@@ -241,12 +230,8 @@ func (r *TreeRepository) getTreeClusterByTreeID(ctx context.Context, treeID int3
 	return tc, nil
 }
 
-// Map sensor, images and tree cluster entity to domain tree
+// Map sensor and tree cluster entity to domain tree
 func (r *TreeRepository) mapFields(ctx context.Context, t *entities.Tree) error {
```

Comment:
You don't touch it but why does `mapTreeCluster` return an error if we ignore it anyway?